pkgs/build-support/node/fetch-npm-deps/default.nix

```
@@ -0,0 +1,52 @@
+{ lib, stdenv, rustPlatform, Security }:
+
+rec {
+  fetch-npm-deps = rustPlatform.buildRustPackage {
+    pname = "fetch-npm-deps";
+    version = (lib.importTOML ./Cargo.toml).package.version;
+
+    src = ./.;
```

Should probably use `lib.cleanSource` just to be safe:
```suggestion
    src = lib.cleanSource ./.;
```

---

I went with a slightly different, more specific filter. I may just tack on `lib.cleanSource` since they can be composed just fine, but not sure.